README.md

```diff
@@ -30,10 +30,17 @@ Folder structure:
         └───file_output.txt
 ```
 
+To run the application:
+
+```
+$ pip install requirements.txt
```

Comment:
I'm pretty sure this needs to be:
`pip install -r requirements.txt`

Reply:
Nice catch!